composer.json

```diff
@@ -1,21 +1,27 @@
 {
     "name": "aoe/imgix",
     "type": "typo3-cms-extension",
+    "description": "Provides the Auto Responsive Images feature of imgix called imgix.fluid()",
+    "homepage": "https://github.com/AOEpeople/TYPO3-Imgix",
+    "support": {
+        "issues": "https://github.com/AOEpeople/TYPO3-Imgix/issues"
+    },
+    "license": "GPL-3.0",
     "authors": [
         {
             "name": "AOE GmbH",
-            "email": "dev@aoe.com"
+            "email": "dev@aoe.com",
+            "homepage": "https://www.aoe.com/",
+            "role": "Developer"
         }
     ],
-    "config": {
-        "secure-http": false
-    },
     "require": {
         "php": ">=5.5.0",
```

Comment:
Die Extension fordert PHP >=5.5 aber TYPO3 6.2 läuft auch mit PHP 5.3.7. Da kann mann diese Extension nicht verwenden.

Reply:
That I would not change.. TYPO3 6.2 is running with 5.5 and therefore we still support TYPO3 6.2.